cocotb/config.py

```diff
@@ -156,6 +156,13 @@ def lib_name_path(interface, simulator):
     return library_name_path
 
 
+def findlibpython():
```

Comment:
```suggestion
def _findlibpython():
```
To make this clear this isn't public API